docs/source/examples/predict_bike_trips.ipynb

```diff
@@ -0,0 +1,414 @@
+    "    base_entity_id='trips',\n",
+    "    new_entity_id='gender',\n",
+    "    index='gender',\n",
+    "    make_time_index=False,\n",
+    ")\n",
+    "\n",
+    "es[\"trips\"][\"gender\"].interesting_values = ['Male', 'Female']\n",
+    "es[\"trips\"][\"events\"].interesting_values = ['tstorms']\n",
+    "es.plot()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Calculating the Features\n",
+    "\n",
+    "Now, we can generate features by using a method called Deep Feature Synthesis (DFS). This will automatically build features by stacking and applying mathematical operations called primitives across relationships in an entity set. The more structured an entity set is, the better DFS can leverage the relationships to generate better features. Let’s run DFS using the following parameters:\n",
+    "\n",
+    "* `entity_set` as the entity set we structured previously.\n",
+    "* `target_entity` as stations, since we want to generate features for each station. \n",
+    "* `cutoff_time` as the label times that we generated previously."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "fm, fd = ft.dfs(\n",
+    "    entityset=es,\n",
+    "    target_entity='from_station_id',\n",
+    "    trans_primitives=['hour', 'week', 'is_weekend'],\n",
+    "    cutoff_time=lt,\n",
+    "    cutoff_time_in_index=True,\n",
+    "    include_cutoff_time=False,\n",
+    "    verbose=False,\n",
+    ")\n",
+    "\n",
+    "fm.head()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "There are two outputs from DFS: a feature matrix and feature definitions. The feature matrix is a table that contains the feature values based on the cutoff times from our labels. Feature definitions are features in a list that can be stored and reused later to calculate the same set of features on future data.\n",
+    "\n",
+    "## Machine Learning\n",
+    "\n",
+    "In the previous steps, we generated the labels and features. The final step is to build the machine learning pipeline.\n",
+    "\n",
+    "### Splitting the Data\n",
+    "\n",
+    "We will start by extracting the labels from the feature matrix and splitting the data into a training set and holdout set."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "y = fm.pop('trip_count')\n",
+    "\n",
+    "splits = evalml.preprocessing.split_data(\n",
+    "    X=fm,\n",
+    "    y=y,\n",
+    "    test_size=0.1,\n",
+    "    random_state=0,\n",
+    "    regression=True,\n",
+    ")\n",
+    "\n",
+    "X_train, X_holdout, y_train, y_holdout = splits"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Finding the Best Model\n",
+    "\n",
+    "Then, we run a search on the training set for the best machine learning model."
```

Comment:
I would provide a little more detail - maybe just mention that EvalML is going to use several different methods for predicting the number of trips and will identify the best one.